### ISSUE TITLE
Nerd Font glyphs are super small

### ISSUE THREAD

**Issue by its-19818942118** [microsoft/cascadia-code#741]
### Cascadia family version

2024 latest realese

### Cascadia family variant(s)

Cascadia Code (the version with ligatures), Cascadia Mono (the version without ligatures)

### Font file format(s)

Windows Terminal included version (TTF (variable)), .ttf (variable), .ttf (static), .otf (static), .woff2 (variable), .woff2 (static)

### Platform

```Linux Aurora 6.8.9-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 02 May 2024 17:49:46 +0000 x86_64 GNU/Linux```

### Other Software

Have only used it with two apps so far
VSCode version:
```
Version: 1.89.0
Commit: b58957e67ee1e712cebf466b995adf4c5307b2bd
Date: 2024-05-01T02:08:25.066Z
Electron: 28.2.8
ElectronBuildId: 27744544
Chromium: 120.0.6099.291
Node.js: 18.18.2
V8: 12.0.267.19-electron.0
OS: Linux x64 6.8.9-arch1-1
```
Kitty:
```
kitty 0.33.1 created by Kovid Goyal
```

### What happened?

well so after checking about the caskaydia code update today was super excited. I installed and all good. but then i tried the Nerd fonts one and both in the kitty terminal and VSCode it was super small.
In VSCode: 
![image](https://github.com/microsoft/cascadia-code/assets/87006850/424657e1-b423-4e60-aa3d-127b2e7a5722)
![image](https://github.com/microsoft/cascadia-code/assets/87006850/2d2a0566-c671-4463-8ca8-9f5c4ad5c38b)
 In Kitty terminal:
![image](https://github.com/microsoft/cascadia-code/assets/87006850/bf7cf1f6-b796-42f1-b7e1-01004a49f7de)
kinda disappointed ngl.

**Comment by aaronbell** [comment]
As I've written elsewhere, the icons are set to be the same width as the rest of the font, which is to be expected from a mono font. However, I do understand that there's now several variants of NF with differing sizes / widths, and I intend to support the larger set in time.

**Comment by its-19818942118** [comment]
Cool
But by default why is the nf Small? Keeping it to a regular height and width. Then adding as you go

**Comment by aaronbell** [comment]
Because this is a mono width font and things should be mono width. The "standard" NF are too big for their squares, which can cause rendering bugs in certain environments.

**Comment by its-19818942118** [comment]
True but anyways I can get over this 
Like if it is small yeah ok. Buts ridiculously small. Like I couldn't distinguish between the smybols

**Comment by Antony-Jones** [comment]
I was also excited to see that NF was included in Cascadia Code, and disappointed when I saw the implementation.

I understand the reasoning, and I understand why it was done this way, but I don't agree with the decision. Personally I think that if you didn't want to add Nerd Fonts  in a way consistent with other implementations, you should have simply not implemented it. 

This is worse than not having it at all in my opinion as it creates confusion that will have to be addressed by the Nerd Fonts team and tooling which makes use of Nerd Fonts (e.g. Oh My Posh) to explain why the glyphs don't look right despite selecting the Nerd Font variant.

**Comment by its-19818942118** [comment]
Yeah I was pretty disappointed with kitty due to it's bad way of handling nerd fonts cuz they utf8 hard to handle etc. but am super disappointed by this. Like I use jetbrains cuz it's good. But cant leave jetbrains don't have continuous ligatures like caskaydia code. Like 4= don't break ligatures was super excited for this release a boomer. Really a boomer ruined my days TT_TT

**Comment by drandarov-io** [comment]
It'd be great if there were two different variants same as with the original Nerd Font / "Cascadia Cove" release.

**Comment by its-19818942118** [comment]
> It'd be great if there were two different variants same as with the original Nerd Font / "Cascadia Cove" release.

that would be nice. or c-c could just create two version. stable and experimental. and add normal glyphs testing instead of small ones.

**Comment by TyCmd** [comment]
I have to agree as I tried it out and thought it was too small as I use it in Wezterm. So, I went back to Caskaydia Cove Nerd Font for now. I hope it gets fixed, as I was excited about this feature being included.

**Comment by mdtauk** [comment]
Would setting these iconic/visual glyphs to use two spaces instead of the single space, keep text aligned to the cell grid, whilst allowing for larger icons aligned to the baseline?

**Comment by mfgbhatti** [comment]
**Cascadia family version**
2024 latest realese

**Cascadia family variant(s)**
`Cascadia Code`
`Caskaydia Cove NF`

**Font file format(s)**
`ttf`

**Platform**
`Linux archlinux 6.6.52-1-lts #1 SMP PREEMPT_DYNAMIC Wed, 18 Sep 2024 19:02:04 +0000 x86_64 GNU/Linux`

**Other Software**
`kitty 0.36.3 created by Kovid Goyal`
`Powerlevel10k v1.20.0`

**What happened?**
Icon size is smaller in Cascadia Code as compared to Caskaydia Cove NF

- With Cascadia code

![Screenshot From 2024-09-26 09-43-51](https://github.com/user-attachments/assets/85a10a50-625e-4d8b-861f-476c665d7477)

- With Caskaydia Cove NF

![Screenshot From 2024-09-26 09-42-52](https://github.com/user-attachments/assets/a5739843-6eda-4ff5-9855-85f01934ef96)
